From ccc1cd5065af27224c37871607e05bca9be380fa Mon Sep 17 00:00:00 2001
From: Anuj Mittal <anujx.mittal@intel.com>
Date: Fri, 11 Mar 2016 10:03:51 +0800
Subject: [PATCH 1/2] security_flags.inc: blacklist more recipes

Add recipes that don't work with the security flags enabled.

Signed-off-by: Anuj Mittal <anujx.mittal@intel.com>
---
 .../meta-oe/conf/distro/include/meta_oe_security_flags.inc            | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/meta-openembedded/meta-oe/conf/distro/include/meta_oe_security_flags.inc b/meta-openembedded/meta-oe/conf/distro/include/meta_oe_security_flags.inc
index 2565e32..2846a60 100644
--- a/meta-openembedded/meta-oe/conf/distro/include/meta_oe_security_flags.inc
+++ b/meta-openembedded/meta-oe/conf/distro/include/meta_oe_security_flags.inc
@@ -5,3 +5,7 @@ SECURITY_CFLAGS_pn-rrdtool = "${SECURITY_NO_PIE_CFLAGS}"
 # This has text reloc errors with the pie options enabled
 SECURITY_CFLAGS_pn-mozjs = "${SECURITY_NO_PIE_CFLAGS}"
 SECURITY_CFLAGS_pn-openldap = "${SECURITY_NO_PIE_CFLAGS}"
+
+SECURITY_CFLAGS_pn-krb5 = "${SECURITY_NO_PIE_CFLAGS}"
+SECURITY_CFLAGS_pn-glog = "${SECURITY_NO_PIE_CFLAGS}"
+SECURITY_CFLAGS_pn-imagemagick = "${SECURITY_NO_PIE_CFLAGS}"
-- 
1.9.1

